src/lib/pipeline_configuration.py

```diff
@@ -45,7 +45,7 @@ class CodeSchemes(object):
 
 class CodingPlan(object):
     def __init__(self, raw_field, coded_field, coda_filename, cleaner=None, code_scheme=None, time_field=None,
-                 run_id_field=None, icr_filename=None, analysis_file_key=None, id_field=None,
+                 run_id_field=None, icr_filename=None, analysis_file_key=None, id_field=None, ws_code=None,
```

Comment:
ws_code or ws_codeID? If I understand correctly these are the IDs associated with the label in the coda scheme that indicates this target? (Just that sentence itself is slightly absurd)

Reply:
No, these are definitely Code objects.

Reply:
so the code that has the ID associated with the label in the coda scheme